generate-emojis-js.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
What is this file? Should it exist, be a command, or maybe be deleted and ship with just a preset of emojis? 🤔

Reply:
This file grabs the current list of emojis from an API and builds the emojis.js file. Its only needed as part of the package so we can occasionally update the emoji list, therefor I don't think it needs to be (or should be) a command, but included it to make updating easy in the future.